QtAwesome/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+project(QtAwesome)
```

Comment:
No cmake_minimum_required command is present

Reply:
Fixed here: 54c87a4